tests/unit/Parsers/ArrayParserTest.php

```diff
@@ -30,11 +30,10 @@ public function testConstructorAcceptsObject()
         $this->assertEquals($expected, $parser->toArray());
     }
 
-    /**
-     * @expectedException InvalidArgumentException
-     */
     public function testArrayParserThrowsExceptionWithInvalidInputOfEmptyString()
     {
+        $this->expectException(\InvalidArgumentException::class);
```

Comment:
Nit: Might be worth moving this to a use statement and cleaning this up.

```suggestion
        $this->expectException(InvalidArgumentException::class);
```